.git-commit-template

```diff
@@ -5,5 +5,38 @@ Explanation
 Resolves:
 https://github.com/SSSD/sssd/issues/XXXX
 
```

Comment:
Seems newline between "Resolves:" and URL prevents github from auto linking PR to issue.

Reply:
For example #5296 wasn't auto linked to #5295

Reply:
I see it linked correctly "This was referenced 11 minutes ago" and link to PR and also this ticket.

Reply:
On the right side of an issue there is
```
Linked pull requests

Successfully merging a pull request may close this issue.
None yet 
```

AFAIR, this field is filled correctly if there is no newline between "Resolves" and URL.

Reply:
I see. We can put it in one line. A commit should fix only one commit anyway.